### PR TITLE
Fixed work. this refs #4

### PR DESCRIPTION
**Pull request by JeanBilheux**

### CODE REVIEW

**Comment by JeanBilheux** [comment]
To check this work, the reviewer can use the IPTS-26894 and one particular run that has the issue

Switch to the right conda environment
`source /opt/anaconda/bin/activate /opt/anaconda/envs/ImagingReduction`

cd to git and this repo
`cd ~/git/NeutronImagingScripts`

add to python path
`export PYTHONPATH=$PWD:$PYTHONPATH`

create tmp output folder
`mkd ~/tmp`

run reduction
`python scripts/mcp_detector_correction.py /SNS/SNAP/IPTS-26894/images/mcp/CT_GS_loopCronTest_10C_Angle_058_723_Goni1_000_000_Goni2_000_000/Run_55098 ~/tmp`

if the tif files show up in the ~/tmp folder, it worked.

File: scripts/mcp_detector_correction.py [hunk 1]
```diff
@@ -40,6 +40,14 @@
     output_dir = args["<output_dir>"]
     skip_first_last_img = args["--skipimg"]
     verbose = args["--verbose"]
+
+    # in some rare instances, the MCP creates a duplicate set of the run in the same folder
+    # we need to only consider the first set in the autoreduction
```
Comment (KedoKudo) [inline scripts/mcp_detector_correction.py:45]: If this is something that occurs very often, we should have a pre-processing to handle the duplicates.
As a quick fix, these changes are okay.
I would recommend writing a new defect/story to address this properly.

File: setup.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 from setuptools import setup, find_packages
 
 NAME = "NeutronImaging"
-META_PATH = os.path.join("neutronimaging", "__init__.py")
+META_PATH = os.path.join("../ToFImaging/jupyter_notebooks/code/neutronimaging", "__init__.py")
```
Comment (KedoKudo) [inline setup.py:8]: This path change seems to be causing issue with the build server.
Please revert the change so that we can get the build to pass.

**Review by KedoKudo — CHANGES_REQUESTED** [review]

**Review by KedoKudo — APPROVED** [review]
